test/README.md

```diff
@@ -35,15 +36,15 @@ This is the preferred method since installing Python 3.4 via Homebrew is no long
 brew update
 brew install pyenv
 
-# Add pyenv init to your shell startup file 
-echo 'eval "$(pyenv init -)"' >> ~/.bash_profile  # replace .bash_profile with whatever you use (.bashrc, .profile, etc.) 
+# Add pyenv init to your shell startup file
+echo 'eval "$(pyenv init -)"' >> ~/.bash_profile  # replace .bash_profile with whatever you use (.bashrc, .profile, etc.)
 
 # Install Python 3.4
 pyenv install 3.4.5  # use latest 3.4.X
```

Comment:
Might as well update this to 3.4.8 until we can move to newer python version.

Reply:
Updated..